fe/lib/hooks/BrowserToken/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface BrowserToken {
+  authorization: string;
+  expiry: string;
+}
```

Comment:
This is the structure that the backend middleware currently uses, but we should probably revisit it. It might be simpler to:

- Pass the original SEEK API structure through the backend
- Simplify this frontend interface so the hook handles expiry and doesn't expose it